language_detector/main.py

```diff
@@ -1,9 +1,27 @@
+# -*- coding: utf-8 -*-
+
 """This is the entry point of the program."""
 
 from .languages import LANGUAGES
 
-
 def detect_language(text, languages=LANGUAGES):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    """Returns the detected lang of given text."""
+    numOfTimesLanguageOccurs = {}
+
+    for word in text.split():
+        lang = whichLanguageIsIt(word, languages)
+        if lang is not None:
+            addToDictionary(lang, numOfTimesLanguageOccurs)
+    return getHighestOccurringLanguage(numOfTimesLanguageOccurs)
+
+def whichLanguageIsIt(word, language_list ):
```

Comment:
Pep 8 suggests to name functions like the main function in this file detect_language. use all lower case words and separate with underscores. Check out https://www.python.org/dev/peps/pep-0008/